src/js/deta.js

```diff
@@ -267,7 +274,7 @@
             close();
             return 200;
         } else {
```

Comment:
We can just return `response.status` outside the conditional, get rid of the `return 200` from the if, and remove the else.